Equilobe.DailyReport.Models/Storage/AtlassianWorklog.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Equilobe.DailyReport.Models.Storage
+{
+    public class AtlassianWorklog
+    {
+        public long Id { get; set; }
+        public long InstalledInstanceId { get; set; }
```

Comment:
The worklog is related to an AtlassianUser, right? Which is already related to an instance. Do we need the InstalledInstanceId here? Do we use this for querying worklogs directly from instance?

Reply:
I decided to have the instanceId here for querying worklogs, to decide if the worklogs retrevied from jira should be added or to be the updates for what we have in db.